dataset/src/test/scala/frameless/functions/NonAggregateFunctionsTests.scala

```diff
@@ -1111,19 +1111,17 @@ class NonAggregateFunctionsTests extends TypedDatasetSuite {
   }
 
   test("Empty vararg tests") {
-    import frameless.functions.aggregate._
     def prop[A : TypedEncoder, B: TypedEncoder](data: Vector[X2[A, B]]) = {
       val ds = TypedDataset.create(data)
       val frameless = ds.select(ds('a), concat(), ds('b), concatWs(":")).collect().run().toVector
-      val framelessAggr = ds.agg(first(ds('a)), concat(), concatWs("x"), litAggr(2)).collect().run().toVector
+      val framelessAggr = ds.agg(concat(), concatWs("x"), litAggr(2)).collect().run().toVector
       val scala = data.map(x => (x.a, "", x.b, ""))
-      val scalaAggr = if (data.nonEmpty) Vector((data.head.a, "", "", 2)) else Vector.empty
+      val scalaAggr = Vector(("", "", 2))
       (frameless ?= scala).&&(framelessAggr ?= scalaAggr)
     }
 
     check(forAll(prop[Long, Long] _))
-    // This fails due to issue #239
-    //check(forAll(prop[Option[Vector[Boolean]], Long] _))
+    check(forAll(prop[Option[Boolean], Long] _))
```

Comment:
Should we keep `Option[Vector[Boolean]]`, maybe as a separate test?

Reply:
I noticed that Vector tests take like 100x to run. I was able to reproduce the problem with a faster alternative. Now that I don't have to run the tests every minute I can probably set it back to be a vector. Let me change that.